src/get-started/java-binding.md

```diff
@@ -543,8 +551,6 @@ For using the library just include the dependency in your `pom.xml`:
   service data (collections and their elements) are available only in a "raw"
   form – without deserialization of the content, which makes their use somewhat
   difficult.
-- [Time oracle](../advanced/time.md) service is not available,
```

Comment:
Woud you get this out of our roadmap too please (as part of https://github.com/exonum/exonum-java-binding/pull/749/)?